UI/newfood.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>fast-food-fast</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/fontawesome.css" integrity="sha384-HbmWTHay9psM8qyzEKPc8odH4DsOuzdejtnr+OFtDmOcIVnhgReQ4GZBH7uwcjf6" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Bree+Serif|Hanalei+Fill" rel="stylesheet">
+</head>
+<body>
+    <article>
+            <!-- Navbar -->
+            <div class="nav">
+                    <div class="nav-header">
+                        <div class="nav-title">
+                        <a href="./index.html">fast-food</a>
+                        </div>
+                    </div>
+                    <div class="nav-btn">
+                        <label for="nav-check">
+                            <span></span>
+                            <span></span>
+                            <span></span>
+                        </label>
+                    </div>
+                    <input type="checkbox" id="nav-check">
+                    <div class="nav-links">
+                        <a href="./index.html" >Home</a>
+                        <a href="./orders.html" >Admin</a>
+                    </div>
+                </div>
+            <br>
+            <br>
+            <br>
+            <br>
+            <br>
+                <!-- sign up -->
+                <form  class="sign-container" >
+                    <div >
+                    <h1>Add Food</h1>
+                    <p>Please input food and Price.</p>
+                    <hr>
+                    <label for="psw"><b>Food</b></label>
+                    <input type="text" placeholder="food" class="food" name="username" required>
+                    
+                    <label for="email"><b>Price</b></label>
+                    <input type="text" placeholder="Enter Price" class="Price" name="Price" required>
+                    <div class="clearfix">
+                        <button type="submit" class="signupbtn">Post Food</button>
+                    </div>
+                    </div>
+                </form>
+                <br>
+            <br>
+            <br>
+            <footer>
+                <h3>fast-food-fast   &copy 2018</h3>
+            
+            </footer> 
+    </article>
+    <script src="./js/jquery.min.js"></script>
+    <script src="./js/auth.js"></script> 
+</body>
+</html>
```

Comment:
#### Nitpick
Insert a new line at the end of file.

Reply:
Please do this for all other files as well. I do not need to comment on every file without a new line at the end.

Reply:
okay